pkgs/tools/graphics/vulkan-tools/default.nix

```diff
@@ -30,7 +30,7 @@ stdenv.mkDerivation rec {
   ];
 
   meta = with stdenv.lib; {
-    description = "LunarG Vulkan loader";
+    description = "Khronos Vulkan Tools";
```

Comment:
> but the github readme says that these are largely written by LunarG.

So why not:

```suggestion
    description = "LunarG Vulkan Tools";
```

Reply:
Probably because that would make confusion between the two worse.